tests/saitests/py3/sai_qos_tests.py

```diff
@@ -2644,6 +2644,13 @@ def runTest(self):
         q_cnt_sum = sum(q_pkt_cnt)
         # Send packets to leak out
         pkt_dst_mac = router_mac if router_mac != '' else dst_port_mac
+
+        is_dualtor = self.test_params.get('is_dualtor', False)
+        def_vlan_mac = self.test_params.get('def_vlan_mac', None)
+        if is_dualtor and def_vlan_mac != None:
```

Comment:
what if it is a dualtor but doesn't have vlan mac, should drop an error for this case? @XuChen-MSFT

Reply:
Solved. Assert when dual-tor miss default vlan mac address.